lib/collibra/rights_to_use_adapter.ts

```diff
@@ -35,9 +35,9 @@ export const rightsToUseAdapter = (asset: Collibra.AssetWithAttributes): RightsT
       ? {
           createdAt: new Date(termsAttr.createdOn),
           id: termsAttr.id,
-          name: "",
+          name: termsAttr.type.name,
           updatedAt: new Date(termsAttr.lastModifiedOn),
-          value: termsAttr.value as string,
+          value: termsAttr.value ? htmlToPortableText(termsAttr.value) : [],
```

Comment:
`null` if not defined, to be in line with assets endpoint

Reply:
I actually did it like this because of the `description`. Should we change that one as well to `null`?